SAR_Training/English/Master/ISCE3_Sentinel1_RTC.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>Similarly, should we add this to a library?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFOpenSARlab/opensarlab-notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>